.config/git-merge-flow-config.jsonc

```diff
@@ -10,12 +10,8 @@
         "vs17.0": {
             "MergeToBranch": "vs17.3"
         },
-        // Automate opening PRs to merge msbuild's vs17.3 (SDK 6.0.4xx) into vs17.4 (SDK 7.0.1xx until 5/2024, VS until 7/2024)
+        // Automate opening PRs to merge msbuild's vs17.3 (SDK 6.0.4xx) into vs17.6 (VS until 1/2025)
         "vs17.3": {
-            "MergeToBranch": "vs17.4"
```

Comment:
Can you also remove 17.0 and 17.3, since those are also out-of-support?

Reply:
And I think 17.9? Though I'm not 100% sure about that

Reply:
`17.0` and `17.3` are still in support:
https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs
till 2024 November :)

Reply:
17.9 is a good catch though.

Reply:
This is updated thank you @Forgind!

Reply:
That's ironic; I checked the Visual Studio lifecycle and forgot about the SDK 😅